### DISCUSSION TITLE
Is docker swarm mandatory for Dokploy?

### DISCUSSION THREAD

**Discussion by Vincent-HD** [Dokploy/dokploy#682] [Q&A] [answered]
Hi,

I would love to setup Dokploy onto a Truenas server of mine, but Truenas only supports Docker/Docker compose containers, I would like to know if there is any way of running inside a docker compose setup instead of swarm mode ?

Thank you 👍

**Comment by Siumauricio** [comment, marked as answer]
you can use only the compose service to create regular docker containers, but the installation is required to be used with swarm

**Comment by caioluis** [comment]
@Vincent-HD what did you end up going with?

**Comment by Casal0x** [comment]
@Vincent-HD did you achieved?
